### ISSUE TITLE
helm3 unable to install

### ISSUE THREAD

**Issue by jurgenweber** [sstarcher/helm-release#11]
```
$ helm version
version.BuildInfo{Version:"v3.0.0", GitCommit:"e29ce2a54e96cd02ccfce88bee4f58bb6e2a28b6", GitTreeState:"clean", GoVersion:"go1.13.4"}
```

```
$ helm plugin install https://github.com/sstarcher/helm-release --version 0.2.2
Downloading https://github.com/sstarcher/helm-release/releases/download/0.2.2/helm-release_0.2.2_darwin_amd64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   629    0   629    0     0   1434      0 --:--:-- --:--:-- --:--:--  1432
100 3684k  100 3684k    0     0   491k      0  0:00:07  0:00:07 --:--:--  769k
Preparing to install into The Kubernetes package manager

...

Use "helm [command] --help" for more information about a command./plugins/helm-release: name too long
Failed to install helm-release
	For support, go to https://github.com/sstarcher/helm-release.
Error: plugin install hook for "release" exited with error
```

**Comment by jurgenweber** [comment]
https://github.com/sstarcher/helm-release/pull/12

**Comment by sstarcher** [comment]
fixed via #12
